ui/src/components/PulseBindings/index.jsx

```diff
@@ -0,0 +1,136 @@
+import { hot } from 'react-hot-loader';
+import React, { Component, Fragment } from 'react';
+import { withApollo } from 'react-apollo';
+import classNames from 'classnames';
+import { withStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListSubheader from '@material-ui/core/ListSubheader';
+import Tooltip from '@material-ui/core/Tooltip';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import PlusIcon from 'mdi-react/PlusIcon';
+import DeleteIcon from 'mdi-react/DeleteIcon';
+
+@hot(module)
+@withApollo
+@withStyles(theme => ({
+  iconButton: {
+    '& svg': {
+      fill: theme.palette.text.primary,
+    },
+  },
+  plusIcon: {
+    marginTop: 80,
+  },
+  deleteIcon: {
+    marginRight: -15,
+    [theme.breakpoints.down('sm')]: {
+      marginRight: -14,
+    },
+  },
+  inputWrapper: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  },
+  inputList: {
+    flex: 1,
+  },
+  bindingListItem: {
+    paddingTop: 0,
+    paddingBottom: 0,
+  },
+}))
+export default class PulseBindings extends Component {
+  render() {
+    const {
+      pulseExchange,
+      pattern,
+      bindings,
+      classes,
+      onBindingAdd,
+      onBindingRemove,
+      onChange,
+    } = this.props;
+
+    return (
+      <Fragment>
+        <div className={classes.inputWrapper}>
+          <List
+            className={classes.inputList}
+            subheader={
+              <ListSubheader className={classes.subheader}>
+                Bindings
+              </ListSubheader>
+            }>
```

Comment:
"Bindings" and "Schedule" don't seem to have the same styling. "Bindings" seems to be smaller. Maybe we can avoid putting the title inside the component and just have it defined in the hook form view.

![Screen Shot 2019-05-15 at 4 38 52 PM](https://user-images.githubusercontent.com/3766511/57807819-f50dc780-772f-11e9-879d-0aaff0c0eae5.png)

Reply:
Actually issue was with adding the bidings inside, `subheader` class was missing from `PulseBinding` component, due to that is was not rendering with correct styles.

I have added subheader style in the component and now it looks fine.

<img width="1597" alt="Screen Shot 2019-05-16 at 10 02 05 PM" src="https://user-images.githubusercontent.com/4654382/57871141-4ac49b80-7826-11e9-85fe-57afd859598e.png">